tests/test_client.py

```diff
@@ -220,3 +220,21 @@ def test_bulk_request(index_client):
     docs = index_client.bulk_request(dids)
     for doc in docs:
         assert doc.did in dids
+
+
+def test_bulk_get_latest(index_client):
```

Comment:
Can we have a mix of versioned and not versioned? If I recall correctly, the idea behind `skip_null=False` is to also return the max versioned if there isn't an unreleased entry for the document

Reply:
The bulk of the unit tests coverage is in indexd, I don't want to duplicate all of them here. In my opinion, indexclient unit tests should really just care about if the right endpoint is called, but I'm doing more that that just to follow previous tests.